### PR TITLE
Add questions card and styling

### PR DESCRIPTION
**Pull request by Rahaf91**

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+}
+/**............Start styling sections..........*/
+.firstQ {
+  color: rgb(255, 255, 255);
+  position: relative;
+  background-color: rgba(34, 94, 150, 0.596);
+  text-align: center;
+  padding: 20px;
+  border-radius: 10px;
+  border: 0px solid black;
```
Comment (ahohnsen) [inline styles.css:29]: If the border is 0px, then it won't be visible 🤔 I guess, you can delete this line.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
```
Comment (eneben) [inline index.html:6]:
```suggestion
    <title>Quiz App</title>
```

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+  margin-bottom: 40px;
+}
+
+main {
+  max-width: 1200px;
+  margin: auto;
+  padding: 0;
+}
+/**............Start styling sections..........*/
+.firstQ {
```
Comment (ahohnsen) [inline styles.css:22]: Please do not use abbreviations as they might be difficult to understand/read. I suggest to use `.firstQuestion` instead.

Additionally, your styles for the first, second and third question are almost the same. I suggest to use one class for all questions, e.g. `.question`. Then you do not have to repeat yourself. Your code will be shorter and easier to understand.

**Review by ahohnsen — CHANGES_REQUESTED** [review]
Hey Rahaf,

Please have a look at my comments. I added some tips how you can improve your CSS code a bit. :)

File: index.html [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+          <li>#git</li>
+          <li>#html</li>
+          <li>#css</li>
+        </ul>
+      </section>
+      <section class="thirdQ">
+        <span class="material-symbols-outlined bookmarked"> bookmark </span>
+        <p>What is the correct HTML for creating a hyperlink?</p>
+        <button>Show Answer</button>
+        <hr />
```
Comment (eneben) [inline index.html:43]: I would suggest to add an answer to a question, maybe with a p- and/or a code-tag?

File: index.html [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+        <ul class="tags">
+          <li>#git</li>
+          <li>#html</li>
+          <li>#css</li>
```
Comment (eneben) [inline index.html:47]: Is it intentional that you have the same tags everywhere? There are questions from different topics. :thinking:

File: styles.css [hunk 6]
```diff
@@ -0,0 +1,106 @@
+* {
+  box-sizing: border-box;
```
Comment (eneben) [inline styles.css:2]: Andrea suggested me yesterday to add also 
```suggestion
  box-sizing: border-box;
  margin: 0;
```

File: styles.css [hunk 7]
```diff
@@ -0,0 +1,106 @@
...
+  text-align: center;
+  max-width: 1200px;
+  margin: auto;
+  padding: 0;
+  border: 1px solid bisque;
+  box-shadow: 3px 3px 2px;
+  color: rgb(6, 94, 94);
+  margin-bottom: 40px;
+}
+
```
Comment (eneben) [inline styles.css:15]: Is it intentional that you don't have any styling for the header?

File: styles.css [hunk 8]
```diff
@@ -0,0 +1,106 @@
...
+  margin-left: -50px;
+}
+
+.tags li {
+  background-color: rgb(29, 74, 95);
+  border: 1px solid rgb(6, 184, 255);
+  border-radius: 5px;
+  padding: 6px;
+}
+.tags li:hover {
```
Comment (eneben) [inline styles.css:49]: cool ;-)

**Review by eneben — CHANGES_REQUESTED** [review]
I still feel a bit overwhelmed by giving a good review without being able to view the website in the browser. Most of it looks good to me. You really accomplished a lot yesterday(?)!! Excellent! :+1:

**Pull request merged**